Annotato/Annotato/Persistence/PersistenceService/RemotePersistenceService/RemoteAnnotationsPersistence.swift

```diff
@@ -46,13 +46,28 @@ struct RemoteAnnotationsPersistence: AnnotationsPersistence {
         return nil
     }
 
-    func createOrUpdateAnnotation(annotation: Annotation) -> Annotation? {
-        fatalError("RemoteAnnotationsPersistence::createOrUpdateAnnotation: This function should not be called")
+    func createOrUpdateAnnotation(annotation: Annotation) async -> Annotation? {
+        guard let senderId = AnnotatoAuth().currentUser?.uid else {
+            return nil
+        }
+
+        let webSocketMessage = AnnotatoCudAnnotationMessage(
+            senderId: senderId,
+            subtype: .createOrUpdateAnnotation,
+            annotation: annotation
+        )
+        WebSocketManager.shared.send(message: webSocketMessage)
+
+        // We do not get any response for the sender from the websocket
         return nil
     }
 
-    func createOrUpdateAnnotations(annotations: [Annotation]) -> [Annotation]? {
-        fatalError("RemoteAnnotationsPersistence::createOrUpdateAnnotations: This function should not be called")
+    func createOrUpdateAnnotations(annotations: [Annotation]) async -> [Annotation]? {
+        for annotation in annotations {
+            await _ = createOrUpdateAnnotation(annotation: annotation)
```

Comment:
random side note but I didn't know it was possible to write in this order, I always thought it'd be
```suggestion
            _ = await createOrUpdateAnnotation(annotation: annotation)
```
isn't it?

Reply:
Yup, I changed